iotilecore/iotile/core/hw/reports/utc_assigner.py

```diff
@@ -120,6 +120,7 @@ def add_point(self, reading_id, uptime=None, utc=None, is_break=False):
         if uptime is None and utc is None:
             return
 
+        ##FIXME: if uptime is None, this will break
```

Comment:
just double checking -- is this just a note for the future or was it intended to be fixed here?

Reply:
Sorry that was just laziness.  I'm fixing it.